public/controllers/agent/components/register-agent.js

```diff
@@ -406,7 +406,7 @@ export class RegisterAgent extends Component {
       rpmText: `sudo ${this.optionalDeploymentVariables()}yum install ${this.optionalPackages()}`,
       debText: `curl -so wazuh-agent.deb ${this.optionalPackages()} && sudo ${this.optionalDeploymentVariables()}dpkg -i ./wazuh-agent.deb`,
       macosText: `curl -so wazuh-agent.pkg https://packages.wazuh.com/4.x/macos/wazuh-agent-${this.state.wazuhVersion
-        }-1.pkg && sudo launchctl setenv ${this.optionalDeploymentVariables()}sudo installer -pkg ./wazuh-agent.pkg -target /`,
+        }-1.pkg && sudo launchctl setenv ${this.optionalDeploymentVariables()} && sudo installer -pkg ./wazuh-agent.pkg -target /`,
```

Comment:
There are two white spaces between `&& sudo installer` and the rest.
![Captura de pantalla 2021-02-22 a las 10 22 27](https://user-images.githubusercontent.com/46006326/108689486-6ed40700-74f9-11eb-9da9-d4cc5ff48354.png)